app/services/sign_in/token_params_validator.rb

```diff
@@ -4,20 +4,22 @@ module SignIn
   class TokenParamsValidator
     include ActiveModel::Validations
```

Comment:
Given your refactor around how validations are done in service objects, it might make sense to refactor this service at some point down the road for consistency.